Makefile

```diff
@@ -51,7 +51,7 @@ fake_translations: extract_translations dummy_translations compile_translations
 pip-compile: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
 	pip install -q pip-tools
 	pip-compile --upgrade -o requirements/base.txt requirements/base.in
-	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in
+	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in requirements/quality.in
```

Comment:
Yes, it combines all of the listed packages and their dependencies in the generated requirements file.  In this case I found myself wanting to run the assorted linting utilities (like `isort`) on individual files often enough that I figured it was worth having them in the main development virtualenv rather than needing to activate the quality checks tox environment in order to use them.